docs/node_benchmarks.md

```diff
@@ -0,0 +1,351 @@
+# Node Benchmarks for Scylla
+
+Whether you're running performance jobs or investigating an issue, you might
+want to now if the hardware Scylla is running meets you baseline perfromance
+expectations. That's what node benchmarks are for.
+
+### What benchmarks are we running?
```

Comment:
I think it must be written in `present simple` tense:
`Which benchmarks do we run?`
The reason for it is that we do not mean some specific benchmark which runs `now`, but those which we are able to run any time.

Reply:
Good catch - it does sound better in present simple. Fixed.